### ISSUE TITLE
Motion vs Occupancy class on devices

### ISSUE THREAD

**Issue by McGiverGim** [Koenkk/zigbee2mqtt#23770]
### What happened?

I've observed that the "icon" in Home Assistant for some motion detection devices (like the RTCGQ11LM from Aqara) is wrong. There are two "similar" [classes ](https://www.home-assistant.io/integrations/binary_sensor/#device-class) for this kind of devices, `motion` and `occupancy`. For a motion detection device, like Aqara, the correct one must be the first, but it's using the second one.
Here you have a sample:
![image](https://github.com/user-attachments/assets/28e4a351-9697-4e2f-b84d-1b189985022f)

The first one is the Aqara with "occupancy", and the second one has the "motion". I don't know when it changed but some time ago it was ok.

### What did you expect to happen?

To have the correct icon based on the device.

### How to reproduce it (minimal and precise)

Simply look at the icon in Home Assistant for this kind of motion devices.

### Zigbee2MQTT version

1.39.1

### Adapter firmware version

20230507

### Adapter

egony ebyte

### Setup

Addon Home Assistant on HAOS on Raspberry Pi

**Comment by McGiverGim** [comment]
I've found the culprit of that: https://github.com/Koenkk/zigbee2mqtt/pull/22896

It seems it was an intended change. So I suppose it will not be a fix for it.

**Comment by AndreKR** [comment]
Just to be sure that you know: When you open the entity settings there is a "Show as" setting that you can switch to "Motion":

![image](https://github.com/user-attachments/assets/29a2f348-54ff-417b-8cbe-f57cd1bdb250)

This will bring back the running man.

**Comment by McGiverGim** [comment]
Yes, that's the workaround I used, but I continue thinking the correct fix is to change it at the beginning.

**Comment by ste-fle** [comment]
I also noticed this issue when I added a presence sensor to a room that already had a motion sensor and suddenly HA showed presence information twice. 
I now changed this manually as suggested above but in my opinion the current motion sensor implementation is wrong. 
Presence information could be an optional sensor based on the available timeout setting but the main sensor should display motion again.

**Comment by ste-fle** [comment]
Still relevant
